tests/unit/remote_build/test_launchpad.py

```diff
@@ -189,6 +192,41 @@ def test_login(self, mock_login):
             version="devel",
         )
 
+    @mock.patch("launchpadlib.launchpad.Launchpad.login_with")
+    def test_login_connection_issues(self, mock_login):
+        mock_login.side_effect = ConnectionRefusedError()
+        raised = self.assertRaises(errors.LaunchpadHttpsError, self.lpc.login)
+        mock_login.side_effect = TimeoutError()
+        raised = self.assertRaises(errors.LaunchpadHttpsError, self.lpc.login)
```

Comment:
just check for the correct exception, get_brief and get_resolution should be tested in tests.unit.remote_build.test_errors (similar structure to others)

Reply:
Will do.